.swiftlint.yml

```diff
@@ -46,7 +46,6 @@ disabled_rules:
 
 included:
   - Contuity
-  - ContuityTests
```

Comment:
why is this being removed?

Reply:
I think this is part of my other branch.

Basically this is the top level swiftlint yml, I made another one for tests (that has slightly more relaxed rules, like it doesn't complain when you force unwrap in unit tests ... this is only to make certain testing easier)